src/androidTest/java/androidx/view/TextViewTest.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2017 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.view
+
+import android.R
+import android.graphics.drawable.BitmapDrawable
+import android.support.test.InstrumentationRegistry
+import android.widget.TextView
+import androidx.graphics.createBitmap
+import org.junit.Assert
+import org.junit.Test
+
+class TextViewTest {
+    private val context = InstrumentationRegistry.getContext()
+    private val resources = context.resources
+    private val textView = TextView(context)
+    private val resourceId = R.drawable.btn_default
+    private val drawable: BitmapDrawable
+        get() {
+            return BitmapDrawable(resources, createBitmap(10, 10))
+        }
+
+    @Test
+    fun updateCompoundDrawables() {
+        textView.updateCompoundDrawables(start = drawable, end = drawable)
```

Comment:
I would prefer if you created drawables directly in these test methods and then used `assertSame` to assert that the correct drawable is set to the correct position. You can use `ColorDrawable` for an easy one to instantiate.

Reply:
Actually you can even use 0xffff0000.toDrawable() :)